### PR TITLE
translations fix, agendaitem improvements

### PR DESCRIPTION
**Pull request by koskelav**
- fixed bug in translation
- removed dotted border from seatmap guest box
- Edit area now has no header, it is shown separately. 
- Styles get now applied on to item html paragraphs as site base css was leaking through. Cumbersome but works mostly. More elegant option would be external .css file to override default font face
- decisions proposal is not shown
- code improvement

### CODE REVIEW

File: WebAPI/WebComponents/Meeting/src/AgendaItem.js [hunk 1]
```diff
@@ -64,63 +71,30 @@ export default function AgendaItem(props) {
...
-            const nodes = div.childNodes
-            for (var i = 0; i < nodes.length; i++) {
-                if (nodes[i].nodeName == 'H1' || nodes[i].nodeName == 'H2' || nodes[i].nodeName == 'H3' || nodes[i].nodeName == 'H4') {
-                    var b = document.createElement('b')
-                    b.textContent = nodes[i].textContent
+        div.innerHTML = html
+        var section = div.querySelector(".SisaltoSektio") || div
+        var nodes = section.childNodes
+        for (var i = 0; i < nodes.length; i++) {
+            var element = nodes[i].cloneNode(true)
```
Comment (vijupa) [inline WebAPI/WebComponents/Meeting/src/AgendaItem.js:81]: could be const also?

File: WebAPI/WebComponents/Meeting/src/AgendaItem.js [hunk 2]
```diff
@@ -64,63 +71,30 @@ export default function AgendaItem(props) {
...
-            editableDiv.innerHTML = div.querySelectorAll(".SisaltoPaatos")[0] || ""
-            return editableDiv.innerHTML
-        } else {
-            const nodes = div.childNodes
-            for (var i = 0; i < nodes.length; i++) {
-                if (nodes[i].nodeName == 'H1' || nodes[i].nodeName == 'H2' || nodes[i].nodeName == 'H3' || nodes[i].nodeName == 'H4') {
-                    var b = document.createElement('b')
-                    b.textContent = nodes[i].textContent
+        div.innerHTML = html
+        var section = div.querySelector(".SisaltoSektio") || div
```
Comment (vijupa) [inline WebAPI/WebComponents/Meeting/src/AgendaItem.js:78]: const?

File: WebAPI/WebComponents/Meeting/src/AgendaItem.js [hunk 3]
```diff
@@ -64,63 +71,30 @@ export default function AgendaItem(props) {
...
+        var nodes = section.childNodes
```
Comment (vijupa) [inline WebAPI/WebComponents/Meeting/src/AgendaItem.js:79]: const?

File: WebAPI/WebComponents/Meeting/src/AgendaItem.js (hunk 1 above)
Comment (koskelav) [reply WebAPI/WebComponents/Meeting/src/AgendaItem.js:81]: element is changed so has to be variable

Comment (vijupa) [reply WebAPI/WebComponents/Meeting/src/AgendaItem.js:81]: ok

**Review by vijupa — APPROVED** [review]

**Pull request merged**